### ISSUE TITLE
Question about the lambda

### ISSUE THREAD

**Issue by Bikesuffer** [Nota-NetsPresso/BK-SDM#33]
Hi there, 
It's me again, I am curious about whether you guys tried different combination of lambda for feat_loss and out_loss or maybe add a lambda for the task_loss?

From my training process, it seems that the feat_loss contributes most part of the total loss.

**Comment by bokyeong1015** [comment]
Hi, thanks for your inquiry.

For our text-to-image experiments, we simply set the loss weights λ_Task, λ_OutKD, and λ_FeatKD to 1, which was effective in empirical validation without hyperparameter tuning and was used in the experiments of our paper.

In recent trials with BK-SDM-Small and batch size 64, changing λ_FeatKD to {0.25, 0.5, 1, 2, 4} did not affect the final generation scores. However, using different scales like 0.01, 0.1, 10, or 100 hasn't been explored.

It would be interesting to study the effect of different loss weightings.

---
added: some experimental results were as follows:
- recent trials with BK-SDM-Small, batch size 64, changing λ_FeatKD to {0.25, 0.5, 1, 2, 4}
![image](https://github.com/Nota-NetsPresso/BK-SDM/assets/15525982/0a83627d-430a-49f5-8c82-22f952fa6642)

- the ablation study presented in our paper (**v:** loss weight = 1, **x**: loss weight = 0)
![image](https://github.com/Nota-NetsPresso/BK-SDM/assets/15525982/069cc45d-c6fd-4d25-99de-ca11361ef568)

**Comment by Bikesuffer** [comment]
Thanks for the information.
